### PR TITLE
[Idle Detection article] Add Puppeteer testing info

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add Puppeteer testing info.

### CODE REVIEW

**Comment by tomayac** [comment]
@sadym-chromium FYI. And thanks for implementing this!

File: src/site/content/en/blog/idle-detection/index.md [hunk 1]
```diff
@@ -188,6 +188,12 @@ You can see the various options in the video below.
   </figcaption>
 </figure>
 
+### Puppeteer support
+
+As of Puppeteer version&nbsp;5.3.1, you can
+[emulate the various idle states](https://github.com/puppeteer/puppeteer/blob/v5.3.1/docs/api.md#pageemulateidlestateoverrides)
```
Comment (tomayac) [inline src/site/content/en/blog/idle-detection/index.md:194]:
```suggestion
[emulate the various idle states](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pageemulateidlestateoverrides)
```

**Comment by mathiasbynens** [comment]
cc @sadym-chromium

File: src/site/content/en/blog/idle-detection/index.md [hunk 2]
```diff
@@ -188,6 +188,12 @@ You can see the various options in the video below.
   </figcaption>
 </figure>
 
+### Puppeteer support
+
+As of Puppeteer version&nbsp;5.3.1, you can
+[emulate the various idle states](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pageemulateidlestateoverrides)
+to test the API over the DevTools Protocol.
```
Comment (mathiasbynens) [inline src/site/content/en/blog/idle-detection/index.md:195]: Over the DevTools Protocol is an implementation detail that might not be super interesting to people. Maybe omit that and focus on what this means for them? Maybe something like:

```suggestion
to programmatically test how your web app’s behavior changes.
```

**Review by mathiasbynens — APPROVED** [review]

**Review by jpmedley — APPROVED** [review]

**Pull request merged**